streams/src/test/java/org/apache/kafka/streams/processor/internals/SinkNodeTest.java

```diff
@@ -20,52 +20,104 @@
 import org.apache.kafka.common.serialization.Serdes;
 import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.common.utils.Bytes;
-import org.apache.kafka.streams.StreamsConfig;
 import org.apache.kafka.streams.errors.StreamsException;
 import org.apache.kafka.streams.state.StateSerdes;
 import org.apache.kafka.test.MockProcessorContext;
 import org.junit.Test;
 
-import java.util.Properties;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
 
 public class SinkNodeTest {
 
     @Test(expected = StreamsException.class)
```

Comment:
Would you mind "fixing" this test to? i.e., using `@Test(expected = SomeException.class)` is bad form for tests with more than a single line. The exception could potentially happen anywhere and my be masking actual bugs. In these cases it is always better to use the same pattern as below, i.e., 
```
try {
    performExceptionThrowingAction()
    fail("KABOOM! shouldn't get here");
} catch (SomeException e) {
   // pass
}
```